docs/nebula-design-concepts.md

```diff
@@ -0,0 +1,216 @@
+# Data modeling and design concept
+------
+**Nebula Graph** is comprised of four components: storage layer, metadata service, compute layer and client console. In this documentation, we'll discuss data modeling and architecture of Nebula Graph.
```

Comment:
composed
computation layer.
This document will discuss data

Reply:
I prefer 'comprise' here, it means 'to have as parts or members', however, 'compose' means 'to be formed from various things'.